lib/components/Application.jsx

```diff
@@ -1,17 +1,45 @@
 import React, { Component } from 'react';
+import { map, filter } from 'lodash'
+import { BrowserRouter, Match, Miss, Link } from 'react-router';
+import data from '../data.js'
+import Search from './Search'
+import IndividualMountain from './IndividualMountain'
+
 
 class Application extends Component {
   constructor() {
     super();
-  }
+			this.state = {
+				data: [],
+				searchString: ''
+			}
+		}
+
+	componentDidMount() {
+		this.setState({
+			data: data
+		})
+	}
+
+	updateSearch(searchString) {
+		this.setState({searchString: searchString});
+	}
 
 	render() {
+
+		let mountainList = this.state.data.map((d) => {
```

Comment:
I think it is still rendering on other routes because of this code here. I tried to pull this out of the render into a function, and call it in the return down below, but I couldn't get that to work. Any advice you have is appreciated!!